src/serviceWorker/index.ts

```diff
@@ -25,8 +25,8 @@ registerRoute(
   new Route(
     ({ url }) => onDemandURLs.includes('.' + url.pathname),
     new CacheFirst({
-      cacheName: 'media',
-      plugins: [new ExpirationPlugin({ maxEntries: 16 })],
+      cacheName: 'on-demand',
```

Comment:
good idea. btw i found out this won't delete the cache automatically so maybe we figure that out now?